src/components/treemap.tsx

```diff
@@ -1,11 +1,10 @@
-import {
-  treemap,
-  treemapResquarify,
-} from 'd3';
+import { treemap, treemapResquarify } from 'd3';
+import { max } from "d3-array";
```

Comment:
I like that we're starting to use the more specific submodules! Nice!
This does look like Intellij may have brought it in though, in which case, we'll want to 'single-quote' the `'d3-array'` part.